src/snowflake/snowpark/modin/plugin/extensions/series_overrides.py

```diff
@@ -17,6 +17,8 @@
 import pandas as native_pd
 from modin.pandas import DataFrame, Series
 from modin.pandas.base import BasePandasDataset
+from modin.pandas.io import from_pandas
+from modin.pandas.utils import is_scalar
```

Comment:
is is_scalar the only util we imported from modin.panda? I recall pandas util also contain is_scalar, right? do you know why we have a separate one in modin util

Reply:
The modin version first checks if the object is a modin frame/series, since the pandas version of the function may materialize its argument.